### PR TITLE
Adding s390x support for dnsmasq and exec-healthz

### PR DESCRIPTION
**Pull request by Ryan-Dmello**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1946)

### CODE REVIEW

**Comment by Ryan-Dmello** [comment]
I signed it!
I am working for IBM and have signed google corporate CLA, could you please verify.

**Comment by luxas** [comment]
I think this depends on: https://github.com/kubernetes/kubernetes/pull/36050

**Comment by Ryan-Dmello** [comment]
I signed it!

**Comment by luxas** [comment]
@Ryan-Dmello, it doesn't seem like the bot picked that up, can you try again?

cc @foxish on the CLA bot

**Comment by luxas** [comment]
LGTM, thanks! Now the bot seems to work.

@Ryan-Dmello Please rebase your PR

cc @bprashanth @bowei @ixdy  I need some of you to actually push the images

**Comment by bowei** [comment]
@luxas -- I can do it, but this patch seems to be out of date

**Comment by ryan-ibm** [comment]
@luxas -- I have updated the patch with latest changes from master branch

File: exec-healthz/build/build.sh [hunk 1]
```diff
@@ -16,6 +16,11 @@
 
 set -o errexit
 set -o nounset
+# For ubuntu updating bash to avoid illegal instruction error for pipefail
```
Comment (luxas) [inline exec-healthz/build/build.sh]: Is this really required here or was it just that you had an old version of bash yourself?

Can we just remove pipefail? Is it really required?

**Comment by luxas** [comment]
Even though I can lgtm this, I can't push the images, just FYI.
I need some Googler to do that :)

for example: @vishh

File: exec-healthz/build/build.sh (hunk 1 above)
Comment (ixdy) [inline exec-healthz/build/build.sh]: yeah, this is a worrying code smell. if we really do need to install bash, it should be done somewhere else than in this script.

**Review by ixdy — DISMISSED** [review]

**Comment by zhengxilong** [comment]
@Ryan-Dmello do have a newer patch according to the comment from @ixdy ?

**Comment by ryan-ibm** [comment]
Sorry for the late response, I have removed the changes related to build.sh, changes as those changes were related to updating the older version of bash .

**Comment by luxas** [comment]
LGTM

**Comment by luxas** [comment]
@ixdy Please push the s390x image for this now :+1:

**Comment by ixdy** [comment]
Sorry for delay. `gcr.io/google-containers/exechealthz-s390x:81a86332` pushed. (Should this be using the v1.2.0 tag? The Makefile here doesn't seem to match others I've seen.)

**Pull request merged**